clang/include/clang/Sema/Sema.h

```diff
@@ -3825,6 +3843,115 @@ class Sema final : public SemaBase {
 
   void redelayDiagnostics(sema::DelayedDiagnosticPool &pool);
 
+  /// Diagnose mutually exclusive attributes when present on a given
```

Comment:
Where did this section of stuff come from?  At least the first 2 are attribute related and might be better in the Attr.td with a SemaRef, perhaps a few others as well.

Reply:
Making functions accept reference to `SemaBase` is novel, but seem to work fine without negative consequences. I moved the whole block to `Sema/Attr.h`.